tests/test_stt.py

```diff
@@ -1,66 +1,136 @@
-# NEON AI (TM) SOFTWARE, Software Development Kit & Application Framework
```

Comment:
The license change here should be reverted; newer notice was replaced with an older one. This block should match: https://gist.github.com/NeonDaniel/1652cbc98717643db50583d80274bca3#file-header-bsd-3-license